modules/adlooxRtdProvider.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+    Module Name: Adloox RTD Provider
+    Module Type: RTD Provider
+    Maintainer: technique@adloox.com
+
+# Description
+
+RTD provider for adloox.com. Contact adops@adloox.com for information.
+
+In addition to populating the ad server key-value targeting, fetched segments (prefixed with `adl_...`) will also populate [First Party Data](https://docs.prebid.org/features/firstPartyData.html), some examples of the segments as described by the Adloox 'Google Publisher Tag Targeting Guidelines' and where they are placed are:
```

Comment:
Please describe the key-value targeting. Is this an integral part of the service or can it be broken out?

Reply:
Sorry, I do not understand your question. Do not all RTD modules provide key-value targeting? What are you proposing or hinting could be broken out here?